### PR TITLE
Update ru_RU locale and fix en-US 

### PR DESCRIPTION
**Pull request by ppv1979**

### CODE REVIEW

File: lib/pkp [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 0b2f4881475dadb5d11b3c114ca7872d6c041f4e
+Subproject commit ca2ab6a2cdde71a6db8531b74c11a55fc2761a0a
```
Comment (mtub) [inline lib/pkp]: please don't commit submodule updates unless necessary and unless you follow https://pkp.sfu.ca/wiki/index.php?title=Github_Documentation_for_PKP_Contributors#PKP_library_submodule_changes

**Comment by mtub** [comment]
@ppv1979 , thanks for the updates. These are a lot of merge commits. Maybe you could look into rebasing (`git --rebase`) before committing? You could try to configure your local git repository that way (by putting `rebase = true` into the branch section of your git config file).

**Comment by ppv1979** [comment]
@mtub, commit for submodule was removed. I also add translations for the latest changes, and remove templates that were added twice in en-US locale by  "Added review request remind auto email templates for each locale" from George that was committed by @asmecher.

**Comment by mtub** [comment]
@ppv1979, thanks for updating this. I will merge. In the future, a cleaner commit history (avoiding the merge commits) would be nice (I know that merging the changes on our end faster helps too).

Thanks for spotting the code duplication in ed9a539b46, following ac9e5d23d4. @asmecher , only the en_US locale had the duplicate code, no further action required.

**Pull request merged**